nstar/src/main.rs

```diff
@@ -55,10 +55,19 @@ async fn send_request(
     client.request(request).await
 }
 
-async fn create_client(host: &str) -> Result<api::client::Client> {
-    api::client::Client::new(&url::Url::parse(host)?)
-        .await
-        .context(format!("Failed to stablish connection to {}", host))
+/// Tries to connect to northstar indefinitely
+async fn try_connect<W: Write>(output: &mut W, url: &url::Url) -> Result<api::client::Client> {
+    loop {
+        match api::client::Client::new(url)
+            .await
+            .context("Failed to Instantiate client")
```

Comment:
That a client is instantiated is a impl details that's not interesting for the user. I'd change this text to "Failed to connect to {}"